mqttx-web/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mqttx-web_app_1
+      APP_PORT: 80
+
+  app:
+    image: emqx/mqttx-web:v1.11.0@sha256:756a5510257309b8921a453b313e0ed4bb5cd812a341b0a10b84518caba4edbe
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
```

Comment:
These environment variables don't exist for the emqx/mqttx-web container so won't actually result in the container running as user 1000.

What you can do instead is completely remove the `environment` section and instead add:

```yaml
user: "1000:1000"
```

...to run the container as non-root

Reply:
fixed and tested.